tests/integration/test_integration.py

```diff
@@ -132,5 +154,6 @@ def set_ACD(cli_runner):
 def test_boostrap_env(clean_vault, set_ACD):
     env = subprocess.check_output("vault env -p A -p C -- env".split())
 
-    assert b"A=B\n" in env
-    assert b"D=E\n" in env
+    assert b"A_VALUE=B\n" in env
```

Comment:
if we do A=A, do we wet A=B or still A_VALUE ?

Reply:
the prefix only change the path used for the variable name, so we will always have SOMETHING_ATTRNAME.
(We could imagine to select only one attribute)

Reply:
Yes, but having a part mandatory in the name kind of defeats the purpose of the alias feature you contributed.

If I have `a: {"b": "secret"}`, and I want `A=secret`, it feels there should be a way.